tests/pytorch_tests/model_tests/feature_models/qat_test.py

```diff
@@ -134,3 +138,50 @@ def compare(self, ptq_model, qat_ready_model, qat_finalized_model, input_x=None,
             # check quantization didn't change when switching between PTQ model and QAT ready model
             qat_finalized_output = qat_finalized_model(_in).cpu().detach().numpy()
             self.unit_test.assertTrue(np.isclose(np.linalg.norm(qat_finalized_output - qat_ready_output) / np.linalg.norm(qat_ready_output), 0, atol=1e-6))
+
+
+class QuantizationAwareTrainingMixedPrecisionCfgTest(QuantizationAwareTrainingTest):
```

Comment:
Add this in a separate dedicated file "qat_mixed_precision_test" or something like that

Reply:
Also, why is it inheriting from QuantizationAwareTrainingTest? Is it correct to use the same "compare" method? I'm not sure that this is what this test should verify.